src/http/formdata.cr

```diff
@@ -39,7 +39,7 @@ require "./formdata/**"
 #
 # To test the server, use the curl command below.
 #
-# ```
+# ```shell
 # $ curl http://localhost:8085/ -F name=foo -F file=@/path/to/test.file
```

Comment:
Added the **shell** style so that it will not be detected as a **crystal code**.

Reply:
Nice! I'll use `console` 👍